AzureMonitorAgent/ama_tst/modules/logcollector.py

```diff
@@ -11,6 +11,10 @@
 RPM_CMD = "rpm -qi azuremonitoragent"
 PS_CMD = "ps -ef | grep {0} | grep -v grep"
 OPENSSL_CMD = "echo | openssl s_client -connect {0}:443 -brief"
+PS_CMD_CPU = "ps aux --sort=-pcpu | head -10"
```

Comment:
Is the `=` supposed to be here?

If so, also, you can make these a bit cleaner by just having one line `PS_CMD = "ps aux --sort {0} | head -10"`, and then whenever you use the string you can use `PS_CMD.format("-rss")`, which will be identical to `PS_CMD_RSS` for example

Reply:
Ah makes sense, have you validated that these commands work on their own? The OMS log collector isn't 100% perfect, so there might be typos - if the command works better as written, then yeah the way you organized it is best

Reply:
Yes I tested the commands work for AMA